src/config.js

```diff
@@ -1,27 +1,32 @@
-const isUndefined = val => typeof val === 'undefined';
+const invariant = (condition, err) => {
```

Comment:
I've picked up this `invariant` pattern from [Flow](https://flow.org/), since it's a special type-checking tool. Open to using a standard `assert` or `if` statement if these feels too "out there."

Reply:
That's interesting, and no I don't think it's too weird. It's a small file so it's easy enough to see what's going on.

I've been using Typescript a lot at work (Angular 6), so I'm getting to be a big fan of type-checking/strong typing. Will have to check out flow sometime as I've heard good things about it.